request/auth.php

```diff
@@ -14,7 +14,7 @@
                     case 'login':
                         if (!empty($_POST['email']) && !empty($_POST['password'])) {
                             if (FirstAide\Utils::isValidEmail($_POST['email'])) {
-                                $Auth = FirstAide\Authentication::withEmailPassword(
+                                $Auth = FirstAide\Authentication::createInstanceWithEmailPassword(
```

Comment:
@fatimarafiqui I agree that global variables have to be used carefully, but mainly when written to it. Global variables are meant to be read, and separation of logic into separate methods / modules should not have an impact on how we read global variables.